src/Microsoft.ML.TimeSeries/TimeSeriesStatic.cs

```diff
@@ -0,0 +1,269 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.TimeSeriesProcessing;
+using Microsoft.ML.StaticPipe.Runtime;
+using System.Collections.Generic;
+using static Microsoft.ML.Runtime.TimeSeriesProcessing.SequentialAnomalyDetectionTransformBase<System.Single, Microsoft.ML.Runtime.TimeSeriesProcessing.IidAnomalyDetectionBase.State>;
+using static Microsoft.ML.Runtime.TimeSeriesProcessing.SequentialAnomalyDetectionTransformBase<System.Single, Microsoft.ML.Runtime.TimeSeriesProcessing.SsaAnomalyDetectionBase.State>;
+
+namespace Microsoft.ML.StaticPipe
+{
+    /// <summary>
+    /// IidChangePoint static API extension methods.
+    /// </summary>
+    public static class IidChangePointStaticExtensions
+    {
+        private sealed class OutColumn : Vector<double>
+        {
+            public PipelineColumn Input { get; }
+
+            public OutColumn(
+                Scalar<float> input,
+                int confidence,
+                int changeHistoryLength,
+                SequentialAnomalyDetectionTransformBase<System.Single, Microsoft.ML.Runtime.TimeSeriesProcessing.IidAnomalyDetectionBase.State>.MartingaleType martingale,
```

Comment:
>SequentialAnomalyDetectionTransformBase<System.Single, Microsoft.ML.Runtime.TimeSeriesProcessing.IidAnomalyDetectionBase.State> [](start = 16, length = 127)

How can I use an alias here instead of spelling this out?  The "using static" above would normally be enough except that I have an IID MartingaleType and an SSA MartingaleType, so it becomes ambiguous.  I tried doing an alias with the using ("using static IidBase = Microsoft.ML..."), but apparently it is not possible to use an alias with a static using.  Any advice?

Reply:
Named and moved inside namespace as suggested by Ivan and Senja.

---
In reply to: [236394897](https://github.com/dotnet/machinelearning/pull/1656#discussion_r236394897) [](ancestors = 236394897)